### PR TITLE
fix missing closing bracket for build {} section

### PR DESCRIPTION
**Pull request by Theragus**
build section not closed, also wrong indention

### CODE REVIEW

File: website/content/docs/templates/hcl_templates/blocks/build/hcp_packer_registry.mdx [hunk 1]
```diff
@@ -46,9 +46,10 @@ Some nice description about the image which artifact is being published to HCP P
     build_labels = {
       "xcode"   = "11.3.0"
       "version" = "Big Sur"
-  }
+    }
 
   sources = ["source.happycloud.macos"]
+  }
```
Comment (sylviamoss) [inline website/content/docs/templates/hcl_templates/blocks/build/hcp_packer_registry.mdx:52]: This bracket goes before sources as this attribute belongs to the `build` block instead `hcp_packer_registry`
```suggestion
```

File: website/content/docs/templates/hcl_templates/blocks/build/hcp_packer_registry.mdx [hunk 2]
```diff
@@ -46,9 +46,10 @@ Some nice description about the image which artifact is being published to HCP P
     build_labels = {
       "xcode"   = "11.3.0"
       "version" = "Big Sur"
-  }
+    }
```
Comment (sylviamoss) [inline website/content/docs/templates/hcl_templates/blocks/build/hcp_packer_registry.mdx:49]: Not sure if my suggested change will be indented correctly, but the closing bracket for hcp_packer_registry should be here
```suggestion
    }
  }
```

**Review by sylviamoss — APPROVED** [review]
Nice catch! One of the closing brackets is placed in the wrong place though. Besides that, it looks good to me 👍🏼

**Review by Theragus — COMMENTED** [review]
sources should not be in the hcp_packer_registry block

**Comment by azr** [comment]
Thanks !

**Pull request merged**